EduVPN-macOS/Controllers/ServersViewController.swift

```diff
@@ -40,7 +40,7 @@ class ServersViewController: NSViewController {
 
         fetchRequest.predicate = NSPredicate(format: "provider != NIL AND (isParent == TRUE OR parent.isExpanded == TRUE)")
         
-        var sortDescriptors = [NSSortDescriptor]() // TODO: This doesn't make much sense
+        var sortDescriptors = [NSSortDescriptor]()
```

Comment:
Only sortName should in theory be enough, and it didn't seem to sort at all…

Reply:
Let’s see what’s up when we tackle the details of sort order. I thought you were referring to this singular line. 👍